pwiz_tools/Skyline/Properties/Settings.cs

```diff
@@ -539,40 +539,22 @@ public List<string> CustomImportTransitionListHeaders
         }
 
         [UserScopedSettingAttribute]
-        // Saves column positions between transition lists. This way when a user tell us the correct column positions they are carried
+        // Saves column type positions between transition lists. This way when a user tell us the correct column positions they are carried
         // on to the next transition list
-        public List<Tuple<int, string>> CustomImportTransitionListColumnsList
+        public List<string> CustomImportTransitionListColumnTypesList
         {
             get
             {
-                if (this[@"CustomImportTransitionListColumnsList"] == null)
+                if (this[@"CustomImportTransitionListColumnTypesList"] == null)
                 {
-                    var list = new List<Tuple<int, string>>();
-                    CustomImportTransitionListColumnsList = list;
-                }
-                return (List < Tuple < int, string>>)this[@"CustomImportTransitionListColumnsList"];
-            }
-            set
-            {
-                this[@"CustomImportTransitionListColumnsList"] = value;
-            }
-        }
-
-        [UserScopedSettingAttribute]
-        public int CustomImportTransitionListColumnCount
-        {
-            get
-            {
-                if (this[@"CustomImportTransitionListColumnCount"] == null)
-                {
-                    var i = new int();
-                    CustomImportTransitionListColumnCount = i;
+                    var list = new List<string>();
+                    CustomImportTransitionListColumnTypesList = list;
                 }
-                return (int)this[@"CustomImportTransitionListColumnCount"];
+                return (List <string>)this[@"CustomImportTransitionListColumnTypesList"];
             }
             set
             {
-                this[@"CustomImportTransitionListColumnCount"] = value;
+                this[@"CustomImportTransitionListColumnTypesList"] = value;
             }

```

Comment:
This should really be one string used three times, rather than three different strings, e.g.
var CustomImportTransitionListColumnTypesList = @"CustomImportTransitionListColumnTypesList";
this avoids the risk of one of the three copies having a typo.
BUT the rest of the file looks this way, and it's better to follow local convention. So this is fine, but yuck.